tests/migration_test.go

```diff
@@ -932,23 +933,45 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 		Context("with an Fedora shared NFS PVC, cloud init and service account", func() {
 			var pvName string
 			var vmi *v1.VirtualMachineInstance
-			BeforeEach(func() {
+
+			initNFS := func(isIpv6 bool) {
 				tests.SkipNFSTestIfRunnigOnKindInfra()
 				pvName = "test-nfs" + rand.String(48)
 				// Prepare a NFS backed PV
 				By("Starting an NFS POD")
 				os := string(cd.ContainerDiskFedora)
-				nfsIP := tests.CreateNFSTargetPOD(os)
+				nfsIPs := tests.CreateNFSTargetPOD(os)
 				// create a new PV and PVC (PVs can't be reused)
 				By("create a new NFS PV and PVC")
+				nfsIP := libnet.GetIp(libnet.GetPodIpsStrings(nfsIPs), isIpv6)
+				if nfsIP == "" && isIpv6 {
+					Skip("skipping ipv6 nfs test on a single stack cluster")
```

Comment:
Might want to make this message unambiguous:
"Skipping IPv6 NFS test, no IPv6 address (might be a single stack IPv4 clutser)"
since single stack IPv6 is a thing we dealt with until recently.

Reply:
This was changed to use a pre-defined [skip](https://github.com/kubevirt/kubevirt/pull/4156/files#diff-e05258dab4f2bda8138464ec283b1214R51). Are you ok with it?